README.md

```diff
@@ -11,10 +11,26 @@ For a comprehensive overview of LLDB, and how Chisel complements it, read Ari Gr
 brew update
 brew install chisel
 ```
+Then check if a  _~/.lldbinit_ file exists in your home directory by writing on the terminal the command below
 
-Then follow the instructions that Homebrew displays to add chisel to your _~/.lldbinit_.
+ ```shell
+ls ~/.lldbinit
+```
+if such file doesn't exist you can create it & open it by tapping on the terminal
+
+ ```shell
+ touch .lldbinit //creates file
+ open .lldbinit //opens file
```

Comment:
If someone doesn't know these commands, they might also not realize that the `//` comments are not supposed to be written. Generally I prefer not adding comments at the end of command lines, but if you feel strongly then they should at least be commented with `#` which is the shell syntax for comments.

Reply:
Oups my bad there! Yeah makes sense.

Reply:
I think the commands are self explanatory so I dropped the comments there. Good point